src/site/paragraphs/table.drupal.liquid

```diff
@@ -27,31 +27,38 @@
     }
 {% endcomment %}
 
-
-<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table">
-
-    {% if entity.fieldTable.caption %}
-        <caption>{{ entity.fieldTable.caption }}</caption>
-    {% endif %}
-
-    {% for value in entity.fieldTable.value %}
-        {% if forloop.first == true %}
-            <thead>
-                <tr>
-                    {% for column in value %}
-                        <th>{{ column }}</th>
-                    {% endfor %}
-                </tr>
-            </thead>
-        {% elsif forloop.last == false %}
-            <tbody>
-                <tr class="class">
+<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table va-table--responsive" role="table">
+    <tbody>
+        {% if entity.fieldTable.caption %}
+            <caption>{{ entity.fieldTable.caption }}</caption>
+        {% endif %}
+        {% assign colLabels = entity.fieldTable.value.0  %}
+        {% for value in entity.fieldTable.value %}
+            {% if forloop.first == true %}
+                <thead>
```

Comment:
I think rendering the `thead` as a child of `tbody` is technically invalid HTML. 

>4.9.1 The `table` element - Content Model: In this order: optionally a `caption` element, followed by zero or more `colgroup` elements, followed optionally by a `thead` element, followed by either zero or more `tbody` elements or one or more `tr` elements, followed optionally by a `tfoot` element, optionally intermixed with one or more script-supporting elements. - https://html.spec.whatwg.org/multipage/tables.html#the-table-element

>4.9.5 The `tbody` element - Content Model: Zero or more `tr` and script-supporting elements. - https://html.spec.whatwg.org/multipage/tables.html#the-tbody-element

Reply:
addressed this feedback as well :)